packages/ui/nuxt.config.js

```diff
@@ -196,7 +202,8 @@ export default {
   content: {},
   i18n: {
     // add SEO attributes in layout head for better performance
-    seo: false,
+    baseUrl: 'https://feeds.witnet.io',
+    seo: true,
```

Comment:
When we implemented this, in the `i18n` docs said it was better to add the `seo` attributes in the [head's layout](https://github.com/witnet/data-feeds-explorer/blob/main/packages/ui/layouts/default.vue#L25) instead of here to have better performance. Could you double check this and remove the comment above if it has changed?